doc/CONFIGURATION.md

```diff
@@ -279,28 +279,94 @@ To increase the maximum object size for writes, use the `--write-part-size` comm
+
+#### Example
+
+```
+s3://amzn-s3-demo-bucket/example-prefix/ /mnt/mountpoint mount-s3 _netdev,nosuid,nodev,nofail,rw 0 0
+```
+The options given mean the following:
+
+* `_netdev` specifies that the filesystem requires networking to mount.
+* `nosuid` specifies that the filesystem cannot contain set userid files.
+* `nodev` specifies that the filesystem cannot contain special devices.
+* `nofail` specifies that failure to mount the filesystem should still allow the system to boot.
+* `rw` specifies that the mount point be created with read and write permissions. Alternatively, use `ro` for read only.
+
+> [!WARNING]
+> The _`netdev`, `nosuid`, and `nodev` options are required when using Mountpoint from fstab. If you do not include these options, Mountpoint will fail to start. We highly recommend you also use the `nofail` option to allow the operating system to start up in case of problems.
+
+> [!IMPORTANT]
+> When running using fstab, Mountpoint will run as root and will use credentials as normal when launched from fstab, but must be available at instance startup. We recommend using IMDS as a credential provider when using Mountpoint with fstab. Given Mountpoint runs as root when using fstab, it will look in root’s home directory for any AWS profile (typically `/root/.aws/config` and `/root/.aws/credentials`).
+
+The options field takes regular Mountpoint CLI arguments as comma separated values in the form `key=value`. For example if you previously used `--allow-delete --uid 7` as CLI arguments you would use `allow-delete,uid=7` in your fstab file.
+
+If you need to include commas in your CLI argument, for example `--cache /tmp/foo,bar`, you would escape the commas in the argument with backslashes like so: `cache=/tmp/foo\,bar`. Characters that need backslash escaping include backslashes (`\`), commas (`,`), and double quotes (`"`).
+
+To stop Mountpoint from running at boot, remove the corresponding line from your fstab file. The filesystem will no longer be automatically mounted on subsequent reboots.
+
+#### Validating changes to the fstab file
+
+If your fstab file is invalid, your operating system may fail to boot. We recommend making a backup of your fstab file before changes, and also using the `nofail` option to ensure invalid Mountpoint configuration or startup failures do not prevent the instance from booting.
+To validate changes, you can use the following snippet, changing `/mnt/mountpoint` to the local path specified in your fstab file:
+
+```
+MNT_PATH=/mnt/mountpoint
+sudo systemctl daemon-reload
+sudo systemctl restart "$(systemd-escape --suffix=mount --path $MNT_PATH)"
+sudo systemctl status "$(systemd-escape --suffix=mount --path $MNT_PATH)"
+```
+
+#### Example output for a successful mount:
```

Comment:
I'd drop the heading to a paragraph. This is just the valid output of the previous section.